lms/static/certificates/sass/_layouts.scss

```diff
@@ -0,0 +1,324 @@
+// ------------------------------
+// // Open edX Certificates: Layouts
+
+// About: styling for generic layouts and responsive support.
+
+// #BASE
+// #ACCOMPLISHMENT
+
+// ------------------------------
+// #BASE
+// ------------------------------
+%layout-wrapper {
+    margin-bottom: spacing-vertical(base);
+    padding: 0 (gutter()*2);
+
+    @include susy-breakpoint($bp-medium, $susy) {
+        padding: 0 gutter();
+    }
+}
+
+%layout {
+    @include container();
```

Comment:
NOTE: new susy grid-management syntax. ``container()`` sets up the parent of columns and its properties (max-width, etc.)